src/jiti.ts

```diff
@@ -175,7 +175,7 @@ export default function createJITI(
     for (const ext of _additionalExts) {
       resolved =
         tryResolve(id + ext, options) ||
-        tryResolve(id.replace(/(c|m)?j(sx?)/, "$1t$2"), options) ||
+        tryResolve(id.replace(/(c|m)?j(sx?)$/, "$1t$2"), options) ||
```

Comment:
Can regex be more specific with dot ext?

Reply:
Done